src/components/thirdparty/mailchimp/VolunteerSignupForm.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import "./VolunteerSignupForm.css";
+import s from "./VolunteerSignupForm.module.css";
+
+/* eslint-disable react/no-danger */
+// This entire fire is just about embedding an external form
```

Comment:
This entire "file"?

Reply:
Good catch! (Though the file kind of looks like a fire with all the inline HTML) 🔥 Will fix it now.